src/Common/tests/System/Net/Configuration.Http.cs

```diff
@@ -14,7 +14,7 @@ public static partial class Http
 
             public static string SecureHost => GetValue("COREFX_SECUREHTTPHOST", DefaultAzureServer);
 
-            public static string Http2Host => GetValue("COREFX_HTTP2HOST", "http2.akamai.com");
+            public static string Http2Host => GetValue("COREFX_HTTP2HOST", "www.microsoft.com");
```

Comment:
For now, I've added a new `Http2NoPushHost` server entry.

Also I've opened a tracking bug about this in general #10958